aspnet-core/xml/ns-Microsoft.AspNetCore.Authentication.Certificate.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Authentication.Certificate">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides classes to enable certificate authentication.</summary>
```

Comment:
Your version is fine, feel free to disregard my suggestion. You can do cert auth without this class, but that class makes it easier.

```suggestion
    <summary>Provides classes to support certificate authentication.</summary>
```